tcms/testruns/static/testruns/js/get.js

```diff
@@ -383,7 +383,7 @@ function bindEvents(selector) {
 function getExpandArea(testExecution) {
     const container = $(`.test-execution-${testExecution.id}`)
 
-    container.find('.test-execution-information .run-date').html(testExecution.close_date || '-')
+    container.find('.test-execution-information .run-date').html(testExecution.stop_date || '-')
```

Comment:
This will very likely fail b/c of other conflicting changes in the API. It's best if you focus on the backend changes first and leave this for later. We may need to push this change to a different branch.